guides/common/assembly_configuring-puppet.adoc

```diff
@@ -0,0 +1,27 @@
+ifdef::context[:parent-context-of-assembly_configuring-puppet: {context}]
+
+ifndef::context[]
+[id="assembly_configuring-puppet"]
+endif::[]
+ifdef::context[]
+[id="assembly_configuring-puppet_{context}"]
+endif::[]
+= Configuring Puppet
+
+:context: assembly_configuring-puppet
+
+[role="_abstract"]
+Puppet is an optional component in {ProjectX}.
+If you installed {Project} afresh and you want to continue using Puppet, you have to enable it in your {Project}.
```

Comment:
Haven't seen `afresh` in the docs yet, so maybe reword? IMHO we always assume the user has a fresh/plain Foreman installation to begin with.

Reply:
I'm pretty sure in the installation guide (at least up until recently), we went with `A fresh installation of {Project}`

Judging by the community behaviour, I think we definitely have to warn people they need to start from scratch. People love to try and install in pretty ripe situations :)

Reply:
FYI, there will be a second procedure for disabling Puppet after upgrade, so it's also a good for contrasting those two scenarios.